make.go

```diff
@@ -392,16 +392,10 @@ func createGitRepository(debsrc, gopkg, orig string, u *upstream,
 		return "", err
 	}
 
-	if err := runGitCommandIn(dir, "init"); err != nil {
+	if err := runGitCommandIn(dir, "init", "-b", debianBranch); err != nil {
```

Comment:
This option has been added in Git v2.28.0, I'm wondering if it wouldn't be better to instead run `git checkout` later on in order to support older Git version. What do you think?

If the end users only install dh-make-golang through the archive this change won't matter (unstable is: 2.32.0), but I'm a bit concerned.

Reply:
> This option has been added in Git 2.28.0

Good catch, I completely missed that! Thanks for checking.

> I'm wondering if it wouldn't be better to instead run git checkout later on in order to support older Git version

So you mean running `git init && git checkout -b <debian-branch>` right? Seems like it's equivalent indeed, let me update the patch.

Reply:
Done. I commented the code with "init -b", so that in the future if we want to use it, it's just a matter of un-commenting it.